charts/gateway-helm/values.tmpl.yaml

```diff
@@ -56,6 +56,7 @@ deployment:
     labels: {}
     topologySpreadConstraints: []
     tolerations: []
+  nodeSelector: {}
```

Comment:
should live at the same level/identation as `topologySpreadConstraints`

Reply:
fixed. thanks